pkgs/tools/networking/ndn-tools/default.nix

```diff
@@ -8,32 +8,22 @@
 , pkg-config
 , sphinx
 , wafHook
+, dpkg
 }:
 
 stdenv.mkDerivation rec {
   pname = "ndn-tools";
-  version = "22.12";
+  version = "24.07";
 
   src = fetchFromGitHub {
     owner = "named-data";
     repo = pname;
     rev = "ndn-tools-${version}";
-    sha256 = "sha256-28sPgo2nq5AhIzZmvDz38echGPzKDzNm2J6iIao4yL8=";
+    sha256 = "sha256-rzGd+8SkztrkXRXcEcQm6rOtAGnF7h/Jg8jaBb7FP9w=";
   };
 
-  # Hacky workaround for new pcap-config.
-  postPatch = ''
-    patch -p1 <<EOF
-      --- a/tools/dump/wscript
-      +++ b/tools/dump/wscript
-      @@ -5 +5 @@
-      -    conf.check_cfg(package='libpcap', uselib_store='PCAP',
-      +    conf.check_cfg(package="", uselib_store='PCAP',
-    EOF
-  '';
-
   nativeBuildInputs = [ pkg-config sphinx wafHook ];
-  buildInputs = [ libpcap ndn-cxx openssl ];
+  buildInputs = [ dpkg libpcap ndn-cxx openssl ];
```

Comment:
Why is this workaround no longer required? Does it have something to do with `dpkg`?

Reply:
They fixed the package configuration in upstream, the same way we would change it with the patch.

Reply:
Dpkg is optional

Reply:
DPKG removed